index.js

```diff
@@ -0,0 +1,31 @@
+const smile = ['&#129313;', '&#128125;', '&#128169;', '&#128123;', '&#128513;'];
+
+const span = document.createElement('span');
+document.body.appendChild(span);
+
+const votes = [0, 0, 0, 0, 0];
+
+const count = () => {
+    for (let i = 0; i < smile.length; i++) {
+        const div = span.querySelector(`.div${i + 1}`);
+        div.innerHTML = `${smile[i]} ${votes[i]}`;
+    }
+};
+
+const clickElem = (index) => {
+    votes[index]++;
+    count();
+};
+
+const divCreate = () => {
+    for (let i = 1; i <= smile.length; i++) {
+        const div = document.createElement('div');
+        span.appendChild(div);
+        div.classList.add(`div${i}`);
+        div.innerHTML = `${smile[i - 1]}`;
+        div.addEventListener('click', () => clickElem(i - 1));
```

Comment:
In general good

- code can be reduced
- adding listener for each smile element, but we need single listener for all

Reply:
Look at this solution

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Voting smiles</title>
    <style>
        .parent {
            display:flex;
            margin: 30px 0 0;
            justify-content: center;
            user-select: none;
        }

        .emoji {
            padding: 10px;
            font-size: 60px;
        }

        .emoji__picture {
            cursor: pointer;
        }

        .emoji__count {
            text-align: center;
        }
    </style>
</head>
<body>
    <div class="parent">
        <div class="emoji">
            <div class="emoji__picture">&#129315;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128578;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128528;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128577;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128557;</div>
            <div class="emoji__count">0</div>
        </div>
        <!-- UNCOMMENT THIS CODE -->
        <!--<div class="emoji">
            <div class="emoji__picture">&#129488;</div>
            <div class="emoji__count">0</div>
        </div>-->
    </div>
    <script>
      const parent = document.querySelector(".parent");

      parent.addEventListener("click", (event) => {
        if (event.target.closest(".emoji__picture")) {
          const nextElementSibling = event.target.nextElementSibling;

          nextElementSibling.textContent = Number(nextElementSibling.textContent) + 1;
        }
      });
    </script>
</body>
</html>
```